### PR TITLE
Server tools API v3.1

### PR DESCRIPTION
**Pull request by Cyberboss**
To go hand in hand with TGS 3.1

- /datum/chat_command for creating commands from in-game
- No more broken relays
- Moves new game announcement to after initializations are complete
- Removes HttpsGet
- Procs for getting the service version and API version

### CODE REVIEW

**Comment by Iamgoofball** [comment]
#31000

**Comment by Cyberboss** [comment]
Omfg i didn't even try

File: code/modules/admin/chat_commands.dm [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	name = "adminwho"
+	help_text = "Lists administrators currently on the server"
+	admin_only = TRUE
+
+/datum/server_tools_command/adminwho/Run(sender, params)
+	return ircadminwho()
+
+/datum/server_tools_command/notify
+	name = "notify"
+	help_text = "Pings the invoker when the running round ends"
```
Comment (Cyberboss) [inline code/modules/admin/chat_commands.dm]: @Shadowlight213 you wanted this?

Comment (Shadowlight213) [reply code/modules/admin/chat_commands.dm]: Nbot already has this. I just need to change the trigger wording a bit to fix it.
Mso wanted something different where the notify would work even if the relay/bots went down, which I know how to do but haven't gotten around to due to various reasons. Adding this to the server tools leaves it at the mercy of the relay working, which doesn't really do what mso wanted.

Comment (Cyberboss) [reply code/modules/admin/chat_commands.dm]: Relays never break with the new version

**Comment by Iamgoofball** [comment]
@Cyberboss generate credits with the server tools when updating from git

just do it

**Comment by Cyberboss** [comment]
It's not something to be hard coded into the server tools, I have another plan for it, but I need us to be running the latest version first

**Comment by Iamgoofball** [comment]
ok

File: code/modules/admin/chat_commands.dm (hunk 1 above)
Comment (Shadowlight213) [reply code/modules/admin/chat_commands.dm]: Oh boy. Screenshotting this for when they do.
Anyways still doesn't do what mso wanted, and is kinda redundant.

**Pull request merged**